server/service/transaction_service.rs

```diff
@@ -733,6 +735,14 @@ impl TransactionService {
                 transaction_options,
             );
             self.transaction = Some(Transaction::Schema(transaction));
+
+            let message_ok_done =
```

Comment:
We need to return the transaction into `self` before exiting this method. Otherwise, the transaction is not closed, and the schema lock is not released....